File: data/CodeLlama/readable_format_HPEC/fortran/file013.f90
program DRB074_flush_orig_yes
    use omp_lib
    use DRB074
    implicit none

    integer :: i, sum
    i = 0
    sum =  0

    !$omp parallel reduction(+:sum) num_threads(10)
    call f1(i)
    sum = sum+i
    !$omp end parallel

    if (sum /= 10) then
        print*,'sum =',sum
    end if
end program

File: data/gpt-4o/readable_format_HPEC/cpp/file013.cpp
#include <stdio.h>
#include <omp.h>

int main()
{
 int i = 0;
 int sum = 0;
#pragma omp parallel reduction(+:sum) num_threads(10)
 {
    f1(&i);
    sum+=i;
 }
 if (sum != 10)
 printf ("sum=%d\n", sum);
 return 0;
}
